Model/Source/DefaultOptions.php

```diff
@@ -17,6 +17,7 @@
 use Magento\Sales\Model\Order;
 use MyParcelNL\Magento\Helper\Checkout;
 use MyParcelNL\Magento\Helper\Data;
+use MyParcelNL\Magento\Helper\ShipmentOptions;
```

Comment:
Ik heb niet het idee dat je deze helper hier ook echt gebruikt, of zie ik wat over het hoofd?

Reply:
Scherp